app/controllers/api/openid_connect/clients_controller.rb

```diff
@@ -1,6 +1,8 @@
 module Api
   module OpenidConnect
     class ClientsController < ApplicationController
+      protect_from_forgery
```

Comment:
I'm wondering if we should add

```ruby
skip_before_action :verify_authenticity_token
```

instead.

If we keep the current line then I think we should describe in a comment why we added this here. Just from the code itself it's not that obvious (at least for me) that this is needed to override the forgery settings from the `ApplicationController` and use `with: :null_session` instead of `with: :exception`.